pages/api/orders/UpdateGoogleSheets.js

```diff
@@ -0,0 +1,34 @@
+import { google } from 'googleapis'; 
+//mport { firebase as fb} from 'googleapis/build/src/apis/firebase';
+//import firebase from '../../../firebase/clientApp';    
+export const config = { // https://nextjs.org/docs/api-routes/api-middlewares
+  api: {
+    bodyParser: true,
+  },
+} 
+
+export default function(firstName, lastName, address, emailAddress, calID, items, deliveryDate) {
```

Comment:
do we need this endpoint? it seems like the addOrder endpoint has it's own copy of this code that it runs rather than hitting this endpoint.

Reply:
No, I added this last second because I thought it might get more complicated when we update three different sheets. Should I just have all the code in AddOrder.js?

Reply:
I'm imagining `AddOrder.js` writes to the 3 different sheets on it's own, otherwise the frontend would have to call both endpoints. Up to you

Reply:
ok, sounds good. I'll do it all in AddOrder.js